system/RESTful/ResourceController.php

```diff
@@ -33,9 +34,9 @@ public function index()
     /**
      * Return the properties of a resource object
      *
-     * @param mixed $id
+     * @param string|null $id
```

Comment:
I'm always torn on Controller parameters. The way you have it is probably the technically-correct solution, but lots of times resource IDs end up coming in as `int` during testing. This change would require casting test values.

Reply:
Changed to `int|string|null`.
We should support int type.